run_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
In my scripts I never use `/usr/bin/env bash`, for two main reasons:
* I've found that `/bin/bash` is more portable (not that we care much about that).
* `/bin/bash` is shorter
* `/bin/bash` allows you to pass default parameters no it, like -x and -e, that can be overriden when invoking the script:
```shell
$ cat myscript.sh
#!/bin/bash -exu
echo lolo
$ bash myscript.sh  # this will ignore the shebang and allow you to override any options passed to bash
$ ./myscript.sh  # this will use the options specified in the shebang
```

The only disadvantage is that if you have a custom bash installation somewhere before on your path (the only time I've seen this is when a machine has been hacked), then it will use that one instead.